databrowser/src/components/pill/linkUtils.ts

```diff
@@ -0,0 +1,8 @@
+export function buildUrlWithQuery(url: string, queryParams: object): string {
```

Comment:
Why is this utils function placed inside the `pill` folder? Isn't it a generic function?

Reply:
Yeah what do you think if we move it into src/domain/datasets? Or where would you put it.

Reply:
Is it related to datasets? I would rather put it somewhere in a shared folder or?